supermarq-benchmarks/supermarq/benchmarks/bit_code.py

```diff
@@ -73,6 +73,14 @@ def circuit(self) -> cirq.Circuit:
 
         return circuit
 
+    def qiskit_circuit(self) -> qiskit.QuantumCircuit:
+        """Generates bit code circuit.
+
+        Returns:
+            A `qiskit.QuantumCircuit`.
+        """
+        raise NotImplementedError("Use circuit() method instead.")
```

Comment:
wdyt about adding this to `supermarq.Benchmark` so we always have a fallback implementation?
```python
    def qiskit_circuit(self) ->  qiskit.QuantumCircuit | list[qiskit.QuantumCircuit]:
        """Returns the qiskit circuit(s) corresponding to the current benchmark parameters."""
        cirq_circuit = self.cirq_circuit()
        if isinstance(cirq_circuit, cirq.Circuit):
            return supermarq.converters.cirq_to_qiskit(cirq_circuit)
        return [supermarq.converters.cirq_to_qiskit(c) for c in cirq_circuit]
```

Reply:
Done.